scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/json/SchedulerJSONUtils.java

```diff
@@ -25,23 +25,15 @@
  */
 package org.ow2.proactive_grid_cloud_portal.scheduler.client.json;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 import org.ow2.proactive_grid_cloud_portal.common.client.json.JSONException;
 import org.ow2.proactive_grid_cloud_portal.common.client.json.JSONUtils;
 import org.ow2.proactive_grid_cloud_portal.scheduler.client.Job;
 import org.ow2.proactive_grid_cloud_portal.scheduler.client.Task;
 import org.ow2.proactive_grid_cloud_portal.scheduler.client.model.JobsPaginationModel;
 
-import com.google.gwt.json.client.JSONArray;
-import com.google.gwt.json.client.JSONNumber;
-import com.google.gwt.json.client.JSONObject;
-import com.google.gwt.json.client.JSONString;
-import com.google.gwt.json.client.JSONValue;
+import com.google.gwt.json.client.*;
```

Comment:
You confirm importing client.* is not >> than importing the 5 ones?

Reply:
I have used the optimize imports from intellij, for me it avoids long lists of imports. I am not sure it has a noticeable impact in memory consumption